docs/src/pages/components/time-picker/SubComponentsTimePickers.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
+import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
+import ClockPicker from '@material-ui/lab/ClockPicker';
+
+export default function SubComponentsTimePickers() {
+  const [date, setDate] = React.useState<Date | null>(new Date());
```

Comment:
What do you think about following the naming convention of the other demos?

```suggestion
  const [value, setValue] = React.useState<Date | null>(new Date());
```

Reply:
See https://github.com/mui-org/material-ui/pull/25812#discussion_r615659468

Reply:
You are right, it seems correct here 👍